cmd/errorutil/internal/config/app.go

```diff
@@ -0,0 +1,6 @@
+package config
+
+const (
+	App     = "errorutil"
+	LogFile = App + ".log"
```

Comment:
Any reason for creating a log file instead of stdout?

Reply:
Yes. Old habit for CLI applications, log to file (also to have history over multiple invocations) and more selective output to console. Don't like getting all log on stdout when using a cli tool. Haven't made up my mind here yet, though, and not finished yet :smile: